<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LAN Bidding System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="Login">
       <widget class="QLabel" name="lblLoginTitle">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>70</y>
          <width>441</width>
          <height>40</height>
         </rect>
        </property>
        <property name="text">
         <string>Chào mừng đến phiên đấu giá</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtUser">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>170</y>
          <width>200</width>
          <height>32</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Nhập tên đăng nhập</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtPass">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>170</y>
          <width>200</width>
          <height>32</height>
         </rect>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::EchoMode::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Nhập mật khẩu</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnLogin">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>230</y>
          <width>200</width>
          <height>40</height>
         </rect>
        </property>
        <property name="text">
         <string>Đăng nhập</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>140</y>
          <width>171</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Tên đăng nhập</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>140</y>
          <width>171</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Mật khẩu</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnOpenRegister">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>230</y>
          <width>200</width>
          <height>40</height>
         </rect>
        </property>
        <property name="text">
         <string>Đăng ký</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="pageLobby">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>211</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Xin chào, [Username]</string>
        </property>
       </widget>
       <widget class="QListWidget" name="listRooms">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>681</width>
          <height>321</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btnCreateRoom">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>430</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Tạo phòng</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnRefresh">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>430</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #27ae60;
}
QPushButton:hover { background-color: #219150; }</string>
        </property>
        <property name="text">
         <string>Làm mới DS</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnJoin">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>430</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Vào phòng</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnHistory">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>430</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Lịch sử</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtSearch">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>10</y>
          <width>301</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Tìm phòng hoặc sản phẩm...</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cboSort">
        <property name="geometry">
         <rect>
          <x>591</x>
          <y>40</y>
          <width>111</width>
          <height>24</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Mới nhất</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Giá tăng dần</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Giá giảm dần</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="btnLogout">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>480</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Đăng xuất</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="pageRoom">
 <layout class="QGridLayout" name="gridLayout_roomRoot">
  <property name="leftMargin"><number>30</number></property>
  <property name="topMargin"><number>20</number></property>
  <property name="rightMargin"><number>30</number></property>
  <property name="bottomMargin"><number>20</number></property>
  <property name="horizontalSpacing"><number>30</number></property>
  <property name="verticalSpacing"><number>15</number></property>

  <!-- LEFT: Auction panel -->
  <item row="0" column="0">
   <layout class="QVBoxLayout" name="vLayout_left">
    <property name="spacing"><number>12</number></property>

    <!-- Header: Item + timer -->
    <item>
     <layout class="QHBoxLayout" name="hLayout_header">
      <item>
       <widget class="QLabel" name="lblRoomName">
        <property name="text">
         <string>Đang đấu giá: </string>
        </property>
        <property name="minimumSize">
         <size><width>260</width><height>0</height></size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_header">
        <property name="orientation"><enum>Qt::Horizontal</enum></property>
        <property name="sizeHint" stdset="0">
         <size><width>40</width><height>20</height></size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdTimer">
        <property name="minimumSize">
         <size><width>220</width><height>60</height></size>
        </property>
        <property name="digitCount"><number>2</number></property>
       </widget>
      </item>
     </layout>
    </item>

    <!-- Info grid -->
    <item>
     <layout class="QGridLayout" name="gridLayout_info">
      <property name="horizontalSpacing"><number>12</number></property>
      <property name="verticalSpacing"><number>10</number></property>

      <item row="0" column="0">
       <widget class="QLabel" name="lblCurrentPrice_2">
        <property name="text"><string>Giá bid hiện tại</string></property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lblCurrentPrice">
        <property name="text"><string>0</string></property>
        <property name="styleSheet"><string>font-weight: 600;</string></property>
       </widget>
      </item>

      <item row="1" column="0">
       <widget class="QLabel" name="labelBuyNowTitle">
        <property name="text"><string>Giá mua ngay</string></property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lblBuyNowPrice">
        <property name="text"><string>0</string></property>
        <property name="styleSheet"><string>font-weight: 600;</string></property>
       </widget>
      </item>

      <item row="2" column="0">
       <widget class="QLabel" name="labelUserCountTitle">
        <property name="text"><string>Số người</string></property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="lblUserCount">
        <property name="text"><string>0</string></property>
       </widget>
      </item>

      <item row="3" column="0">
       <widget class="QLabel" name="labelNextItemTitle">
        <property name="text"><string>SP tiếp theo</string></property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="lblNextItem">
        <property name="text"><string>(Không có)</string></property>
       </widget>
      </item>

      <item row="4" column="0">
       <widget class="QLabel" name="labelTimeTitle">
        <property name="text"><string>Thời gian</string></property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="lblTimeLeft">
        <property name="text"><string>0s</string></property>
       </widget>
      </item>

      <item row="5" column="0">
       <widget class="QLabel" name="labelBidInputTitle">
        <property name="text"><string>Nhập giá</string></property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="txtBidAmount">
        <property name="placeholderText">
         <string>Ví dụ: 120000</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>

    <!-- Action buttons -->
    <item>
     <layout class="QHBoxLayout" name="hLayout_actions">
      <item>
       <widget class="QPushButton" name="btnBid">
        <property name="text"><string>Đặt giá</string></property>
        <property name="styleSheet"><string>background:#3daee9; color:white; padding:6px 14px; border-radius:6px;</string></property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnBuyNow">
        <property name="text"><string>MUA NGAY</string></property>
        <property name="styleSheet"><string>background:#f39c12; color:white; padding:6px 14px; border-radius:6px; font-weight:600;</string></property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_actions">
        <property name="orientation"><enum>Qt::Horizontal</enum></property>
        <property name="sizeHint" stdset="0">
         <size><width>40</width><height>20</height></size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>

    <!-- Room log -->
    <item>
     <widget class="QTextEdit" name="txtRoomLog">
      <property name="readOnly"><bool>true</bool></property>
      <property name="minimumSize">
       <size><width>520</width><height>220</height></size>
      </property>
     </widget>
    </item>

    <!-- Leave -->
    <item>
     <layout class="QHBoxLayout" name="hLayout_leave">
      <item>
       <widget class="QPushButton" name="btnLeave">
        <property name="text"><string>Rời phòng</string></property>
        <property name="styleSheet"><string>background:#e74c3c; color:white; padding:6px 14px; border-radius:6px;</string></property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_leave">
        <property name="orientation"><enum>Qt::Horizontal</enum></property>
        <property name="sizeHint" stdset="0">
         <size><width>40</width><height>20</height></size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>

   </layout>
  </item>

  <!-- RIGHT: Chat panel -->
  <item row="0" column="1">
   <layout class="QVBoxLayout" name="vLayout_chat">
    <property name="spacing"><number>10</number></property>

    <item>
     <widget class="QLabel" name="labelChatTitle">
      <property name="text"><string>=== Chat phòng ===</string></property>
      <property name="alignment"><set>Qt::AlignHCenter|Qt::AlignVCenter</set></property>
     </widget>
    </item>

    <item>
     <widget class="QTextEdit" name="txtChatLog">
      <property name="readOnly"><bool>true</bool></property>
      <property name="minimumSize">
       <size><width>380</width><height>260</height></size>
      </property>
     </widget>
    </item>

    <item>
     <layout class="QHBoxLayout" name="hLayout_chatInput">
      <item>
       <widget class="QLineEdit" name="txtChatInput">
        <property name="placeholderText"><string>Nhập tin nhắn...</string></property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSendChat">
        <property name="text"><string>Gửi chat</string></property>
        <property name="styleSheet"><string>background:#3daee9; color:white; padding:6px 14px; border-radius:6px;</string></property>
       </widget>
      </item>
     </layout>
    </item>

    <item>
     <spacer name="verticalSpacer_chat">
      <property name="orientation"><enum>Qt::Vertical</enum></property>
      <property name="sizeHint" stdset="0">
       <size><width>20</width><height>40</height></size>
      </property>
     </spacer>
    </item>

   </layout>
  </item>

 </layout>
</widget>

     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
